src/binarytree.jl

```diff
@@ -143,3 +143,89 @@ function _printkeyvalue(io::IO, node::BinaryNode)
     show(ioctx, val)
   end
 end
+
+# -----------
+#  UTILITIES
+# -----------
+
+"""
+  BinaryTrees.minnode(tree)
+
+Find the `node` with the smallest `key` in the `tree`.
+
+  BinaryTrees.minnode(node)
+
+Find the `node` with the smallest `key` in the subtree rooted at `node`.
+If `nothing` is provided, `nothing` is returned.
+"""
```

Comment:
```suggestion
"""
  BinaryTrees.minnode(tree)

Find the node with the smallest key in the `tree`.

  BinaryTrees.minnode(node)

Find the node with the smallest key in the subtree rooted at `node`.
If `nothing` is provided, `nothing` is returned.
"""
```
Tip: You only need to reference (with backquotes) symbols that appear in the docstring, such as "node" and "tree", or that are language symbols, such as "nothing".

Reply:
Awesome. I went back and forth with "well they are types that are public and important but also they aren't args sooo.. and just went with the conservative option. Good to know! Committing those right now.